test/dynamicFormControlGroup.test.ts

```diff
@@ -60,43 +60,36 @@ describe('Dynamic form control group component', function () {
         });
     };
 
-    it("renders label if it exists", () => {
+    it("renders conditional label and its contents", () => {
         const rendered = shallowMount(DynamicFormControlGroup, {
             propsData: {
-                controlGroup: fakeFormGroup
+                controlGroup: fakeFormGroup,
+                readonly: true
             }
         });
-
-        const labelCol = rendered.find("label");
+        const conditionalLabel = rendered.find("conditional-label-stub")
```

Comment:
This might be more intuitive:
```suggestion
        const conditionalLabel = rendered.find(ConditionalLabel)
```